src/Symfony/Component/Security/Core/Exception/CustomUserMessageAuthenticationException.php

```diff
@@ -60,11 +60,9 @@ public function getMessageData()
      */
     public function serialize()
     {
-        return serialize([
-            parent::serialize(),
-            $this->messageKey,
-            $this->messageData,
-        ]);
+        return serialize([parent::serialize(true), $this->messageKey, $this->messageData]);
+
+        return $this->doSerialize($serialized, \func_num_args() ? \func_get_arg(0) : null);
```

Comment:
**Problem:** double return!

(seeing the others changes, I guess the first line should be `$serialized = [...];`)

Reply:
thanks @guilliamxavier , see https://github.com/symfony/symfony/pull/30044